tests/test_merge.py

```diff
@@ -0,0 +1,216 @@
+"""Test class for merging multiple yaml files into one."""
+from io import StringIO
+from maraplus.parser import YamlParser
+
+
+YAML_BASE = """
+---
+migration:
+  options:
+    install_command: odoo2
+  versions:
+    - version: setup
+      operations:
+        pre:
+          - echo 'pre-operation'
+      addons:
+        install:
+          - crm
+        upgrade:
+          - note
+    - version: 0.1.0
+      operations:
+        post:
+          - echo 'post-operation'
+      addons:
+        install:
+          - web
+          - contacts
+"""
+
+YAML_CUSTOM_2 = """
```

Comment:
Where is YAML_CUSTOM_1?
```suggestion
YAML_CUSTOM_1 = """
```

Reply:
It means second yml, third yml. First yml is base. But I guess, can number all even base one.